charm/docs/tutorial/test-netbox.md

```diff
@@ -0,0 +1,54 @@
+# Testing NetBox
```

Comment:
I couldn't find it but I think that there is a standard for writing tutorials starting with "What you'll do" like this:
https://github.com/canonical/jenkins-k8s-operator/blob/main/docs/tutorial/getting-started.md
Also, minor: file name should be equal to the title

Reply:
I also couldn't find it, but I think this is correct. It's good practice to have a "What you'll do" section and a prerequisites section for tutorials

Reply:
similar in here: https://diataxis.fr/tutorials/#the-language-of-tutorials

Reply:
Improved the structure a bit!